bedrock/mozorg/redirects.py

```diff
@@ -146,8 +146,6 @@ def decider(request, **kwargs):
     redirect(r'^foundation/identity-guidelines/index.html', 'styleguide.home'),
     redirect(r'^foundation/identity-guidelines/mozilla-foundation.html',
              'styleguide.identity.mozilla-branding'),
-    redirect(r'^foundation/identity-guidelines/thunderbird.html',
```

Comment:
Removing this redirect allows `foundation/identity-guidelines/thunderbird.html` to 404. Is this intentional?

Reply:
Initially this PR included removing the style guide as well but I scaled it back. I guess this is an artifact.

But we are planning to take the style guide out, with it gone I'm not sure there's an appropriate place to redirect this. Ideas?

Reply:
I'll add it back and let the style guide PR deal with it.